src/Tasks/SynTree/Config.hs

```diff
@@ -91,6 +93,9 @@ checkSynTreeConfig SynTreeConfig {..}
       = reject $ do
         english "Your maximum depth value is unreasonably large, given your other settings."
         german "Maximale Tiefe des Baumes ist zu hoch für eingestellte Parameter."
+    | minUniqueBinOperators < 0 = reject $ do
+        english "There should be a positive number of unique operators"
+        german "Es sollte eine positive Anzahl an unterschiedlichen Operatoren geben"
```

Comment:
`positive` -> `non-negative`, da `0` ja erlaubt wird

Reply:
Vermutlich wäre es auch noch sinnvoll, gegen ein zu hohes Setzen von `minUniqueBinOperators` vorzugehen. Also etwa `minUniqueBinOperators == 5` wäre ja problematisch, wenn es überhaupt nur `And`, `Or`, `Impl`, `Equi` gibt.

Die tatsächliche Anzahl von Operatoren im Datentyp (um auf etwaige spätere Erweiterungen vorbereitet zu sein) müsste sich durch einen Ausdruck wie `length [ minBound .. maxBound :: BinOp]` "dynamisch" ermitteln lassen.